.github/ISSUE_TEMPLATE/bug_report.yml

```
@@ -0,0 +1,58 @@
+name: Bug Report
+description: Submit a bug report.
+labels: [bug]
+body:
+  - type: textarea
+    attributes:
+      label: Description
+      description: A clear and concise description of what the bug is, and why you consider it to be a bug.
+    validations:
+      required: true
+  - type: checkboxes
+    attributes:
+      label: Verification
+      description: Please verify that you've followed these steps.
+      options:
+        - label: I have verified that my MSYS2 is up-to-date before submitting the report (see https://www.msys2.org/docs/updating/)
+          required: true
+  - type: textarea
+    attributes:
+      label: Windows Version
+      description: "Example: Windows 10 10.0.19044"
```

I think this would help to get the Windows version.

```suggestion
      description: Please run `cmd.exe /c ver` to get the build of Windows you are on.
      placeholder: "Microsoft Windows [Version 10.0.19042.867]"
```

---

`uname` or `uname -a` would work too

---

```
$ uname
MINGW64_NT-10.0-19044
$ uname -a
MINGW64_NT-10.0-19044 DESKTOP-****** 3.3.5-341.x86_64 2022-07-04 21:33 UTC x86_64 Msys
```